### PR TITLE
[build] Bump Gradle to 5.2

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: PCGen-Formula/gradlew [hunk 1]
```diff
@@ -28,16 +28,16 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```
Comment (karianna) [inline PCGen-Formula/gradlew:31]: need to revert this

File: PCGen-Formula/gradlew.bat [hunk 2]
```diff
@@ -14,7 +14,7 @@ set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=
+set DEFAULT_JVM_OPTS="-Xmx64m"
```
Comment (karianna) [inline PCGen-Formula/gradlew.bat:17]: need to revert this

**Review by karianna — DISMISSED** [review]
Some fixes needed

File: PCGen-Formula/gradlew (hunk 1 above)
Comment (grimreaper) [reply PCGen-Formula/gradlew:31]: why? It is needed for windows and does not affect functioning of the application.

Comment (karianna) [reply PCGen-Formula/gradlew:31]: It's limiting the max memory to 64M - PCGen won't run

Comment (grimreaper) [reply PCGen-Formula/gradlew:31]: nope. It affects gradle, not the application.

**Comment by karianna** [comment]
Original commit is still there?

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**